openmetadata-ui/src/main/resources/ui/src/styles/components/react-awesome-query.less

```diff
@@ -24,6 +24,13 @@
 
   & > .group-or-rule-container.group-container {
     & > .group.group-or-rule {
+      .group--children {
+        .group-or-rule-container.group-container:first-child {
+          .group--header .action.action--DELETE {
+            display: none;
+          }
+        }
+      }
```

Comment:
Do we want this only for the first delete option?

Reply:
Yes, We want to restrict user from deleting one and only group which will leads to uncertain UI